packages/cli/templates/web-library-package/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "@backstage/cli/config/tsconfig.json",
+  "include": [
+    "src",
+    "dev"
```

Comment:
Is there a point to having a `dev` folder?

Reply:
No, it shouldn't! That's a good catch. I'll get it updated in my next commit